price-feeder/config/config.go

```diff
@@ -216,9 +218,17 @@ func ParseConfig(configPath string) (Config, error) {
 		}
 	}
 
+	// Use currencyProviderTracker to check available exchanges for each currency pair
+	// and determine minimum amount of providers required for each pair
+	currencyProviderTracker, err := NewCurrencyProviderTracker(ctx, logger, cfg.CurrencyPairs...)
+	if err != nil {
+		return cfg, fmt.Errorf("failed to start currency provider tracker: %w", err)
+	}
+
```

Comment:
Imo the `ParseConfig` function should be limited to parsing the config file itself and checking against hard-coded rules. Maybe we move this into another function, `CheckProviderMins` or somthing?

Reply:
This code is probably fine in `cmdHandler`

Reply:
Created CheckProviderMins method and I'm calling it in priceFeederCmdHandler now so ParseConfig doesn't need to take the logger and context anymore.